src/custom-types.json

```diff
@@ -476,6 +476,15 @@
   "audio/l24": {
     "compressible": false
   },
+  "audio/mobile-xmf": {
+    "extensions":["xmf", "mxmf"],
+    "sources": [
+      "http://tools.ietf.org/rfc/rfc4723.txt",
+      "http://www.iana.org/assignments/media-types/audio/mobile-xmf",
+      "https://filext.com/file-extension/XMF",
+      "http://www.loc.gov/preservation/digital/formats/fdd/fdd000121.shtml"
```

Comment:
This seems to be a secondary source, especially when the document lists the file extension "XMF" and states that it is "Suggested by the specification". What specification is it referring to as where the XMF file extension comes from?

Reply:
XMF seems to have been created by the MIDI association. [spec](https://www.midi.org/specifications/item/xmf-specification-all). But I could not find a direct mapping between the file extension and the mime-type in their spec though there are references to audio/mobile-xmf. I can remove this as well.